### PR TITLE
fix: css variables consistency updates

### PR DESCRIPTION
**Pull request by gkatsev**
BREAKING CHANGE: Change `--media-live-indicator-color` to `--media-live-button-indicator-color`, `--media-time-buffered-color` to `--media-time-range-buffered-color`, `--media-background-position` to `--media-poster-image-background-position`, and `--media-background-size` to `--media-poster-image-background-size`.

### CODE REVIEW

File: examples/vanilla/components.html [hunk 1]
```diff
@@ -0,0 +1,287 @@
+<!DOCTYPE html>
```
Comment (cjpillsbury) [inline examples/vanilla/components.html:1]: praise: 😍

**Review by luwes — APPROVED** [review]
LGTM 👍

File: src/js/media-text-display.js [hunk 2]
```diff
@@ -56,7 +56,6 @@ template.innerHTML = /*html*/`
  *
  * @cssproperty --media-control-display - `display` property of control.
  * @cssproperty --media-control-background - `background` of control.
- * @cssproperty --media-control-hover-background - `background` of control hover state.
```
Comment (luwes) [inline src/js/media-text-display.js:59]: oh since you deleted here, it's needed in media-time-display

**Review by luwes — CHANGES_REQUESTED** [review]

File: src/js/media-text-display.js (hunk 2 above)
Comment (gkatsev) [reply src/js/media-text-display.js:59]: ah, good catch

Comment (gkatsev) [reply src/js/media-text-display.js:59]: Updated

**Review by cjpillsbury — APPROVED** [review]

**Pull request merged**